src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchTrialDAO.scala

```diff
@@ -121,16 +152,12 @@ class ElasticSearchTrialDAO(client: TransportClient, indexName: String, refreshM
     * @return the updated project record
     */
   override def claimProjectRecord(userInfo: WorkbenchUserInfo): TrialProject = {
-    val nextProjectQuery = boolQuery()
-      .must(termQuery("verified", true))
-      .mustNot(existsQuery("user.userSubjectId.keyword"))
-
     // if we find regular race conditions in which multiple users attempt to claim the "next" project,
     // we could change this to return N (= ~20) available projects, then choose a random project
     // from that list.
     val nextProjectRequest = client
```

Comment:
this still would not keep multiple clients from choosing the same project

Reply:
even if multiple users find the same record from the read query, the update query will only succeed for one of them due to the version checks. So this comment could read something like "if we find that we are regularly retrying this method in order to succeed at claiming a record, we could return 20 and pick a random one from that list, which would lessen the problem but not eliminate it entirely".